### ISSUE TITLE
Add luarocks

### ISSUE THREAD

**Issue by TangentFoxy** [lukesampson/scoop#1592]
I've been trying to figure out how to get LuaRocks to install via scoop but I've utterly failed at it. Could someone smarter give it a try?

**Comment by r15ch13** [comment]
Is `/SELFCONTAINED` useful?
What about the parameters listed [here](https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Windows). Are they necessary during installation or are they optional? I mean the "Lua interpreter" ones.
Does this benefit from making "rocks" [persistent](https://github.com/lukesampson/scoop/wiki/Persistent-data)?

**Comment by Guard13007** [comment]
@r15ch13 `/SELFCONTAINED` puts everything in one directory, I'd prefer to have the rocks in persistent storage. I have attempted several variations of setting up a manifest for LuaRocks using those options and have not had any success, I think due to me not understanding properly how Lua / LuaRocks / scoop / Powershell / command prompt interact. LuaRocks installs using a batch script, which I cannot get to run or have access to the proper variables from scoop in Powershell.

The parameters are optional if I remember correctly, and will be prompted for or set to defaults if not provided. However, this breaks the way scoop is supposed to work. I also want LuaRocks installed in this manner to depend on and work with Lua provided via scoop.

In short, I have tried several variations of having scoop download and run `install.bat` with proper variables to install LuaRocks, and feel completely lost.

**Comment by r15ch13** [comment]
Did you try the `[installer](https://github.com/lukesampson/scoop/wiki/App-Manifests)` property?
```
"installer": {
  "file": "install.cmd",
  "args": "/SELFCONTAINED"
}
```

Also, is this a proper Lua version to use?https://sourceforge.net/projects/luabinaries/files/5.3.4/Tools%20Executables/
It seems the version from the main Bucket is older and the project is [discontinued](https://github.com/rjpcomputing/luaforwindows/commits/master).

**Comment by rasa** [comment]
Implemented, ScoopInstaller/Main@63d2a5cf4f6c274116e8c5af6eb877fe8d02b7fd, ScoopInstaller/Main@99fd298d4ef907cffe7a7297f4d30e702e6b8dfd

**Comment by Vloz** [comment]
> Implemented, [ScoopInstaller/Main@63d2a5c](https://github.com/ScoopInstaller/Main/commit/63d2a5cf4f6c274116e8c5af6eb877fe8d02b7fd), [ScoopInstaller/Main@99fd298](https://github.com/ScoopInstaller/Main/commit/99fd298d4ef907cffe7a7297f4d30e702e6b8dfd)

Sorry new here, are these changes really related to luarocks? @rasa

**Comment by jamesgecko** [comment]
Yeah, don't packages don't appear to expose a `luarocks` binary. `luarocks` is provided by the `lua-for-windows` package, but it fails to install every lua package I've tried. For example:

```
> luarocks install moonscript
Installing http://luarocks.org/repositories/rocks/moonscript-0.5.0-1.src.rock...

Error: Failed unpacking rock file: C:\Users\James\AppData\Local\Temp\luarocks_luarocks-rock-moonscript-0.5.0-1-9253/moonscript-0.5.0-1.src.rock
```
